packages/bruno-app/src/components/Modal/index.js

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from 'react';
 import StyledWrapper from './StyledWrapper';
+import { useRef } from 'react';
```

Comment:
```suggestion
import React, { useEffect, useState, useRef } from 'react';
import StyledWrapper from './StyledWrapper';
```
Since there are react imports already so better to club with it